nservicebus/messaging/message-correlation.md

```diff
@@ -7,14 +7,18 @@ related:
  - nservicebus/messaging/headers
 ---
 
-Message correlation is the act of connecting instances of request messages with instances of response messages. Since all messages have a `Message Id` the `Correlation Id` is just a reference back to the specific message instance that caused this message to be sent back as a reply.
+Message correlation connects instances of request messages with instances of response messages. Each message has a `Message Id` and the `Correlation Id` is a reference back to the specific message instance that caused this message to be sent back as a reply.
```

Comment:
Good questions, if it's not clear to us, how can it be clear to our users?

@MarcinHoppe CorrelationId and MessageId can be the same, if there's no originating message. From what I understand, this is always the case with commands?

Also it's focused on the reply, but isn't the CorrelationId different when publishing events?